users/init.sls

```diff
@@ -13,14 +11,22 @@ extend: {{ datamap.sls_extend|default({}) }}
 {%- endmacro -%}
 
 {% set users = salt['pillar.get']('users:manage', {}) %}
+{% set absent_users = salt['pillar.get']('users:absent', {}) %}
 {% set groups = salt['pillar.get']('groups:manage', {}) %}
+{% set absent_groups = salt['pillar.get']('groups:absent', {}) %}
+
+{% for id, g in absent_groups|dictsort %}
+  {% set name = g.name|default(id) %}
+group_absent_{{ name }}:
+  group.absent:
+    - name: {{ name }}
+{% endfor %}
 
 {% for id, g in groups|dictsort %}
   {% set name = g.name|default(id) %}
 
 group_{{ name }}:
-  group:
```

Comment:
I like the Syntax suggested by the SaltStack community a little more, but in this case it just makes more sense to do it your way.
Thank you.